src/main/java/core/basesyntax/transaction/FruitTransaction.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.transaction;
+
+import core.basesyntax.operation.Operation;
+
+public class FruitTransaction {
```

Comment:
This class could be made immutable by removing the setter methods and making the fields final, which would then be initialized via a constructor. This is a common practice for data transfer objects and can help prevent accidental modification of the object's state.